mermithid/processors/IO/MultiChannelCicadaReader.py

```diff
@@ -0,0 +1,108 @@
+'''
+Processor that can read (not write) Katydid ROOT files using the Cicada library
+Author: C. Claessens
+Date: April 08 2020
+'''
+
+from morpho.processors.IO import IOProcessor
+from morpho.utilities import reader, morphologging
+logger = morphologging.getLogger(__name__)
+
+try:
+    from ROOT import TFile, TTreeReader, TTreeReaderValue
+except ImportError:
+    pass
+
+import numpy as np
+
+class MultiChannelCicadaReader(IOProcessor):
+    '''
+    Processor that can read (not write) Katydid ROOT files using the Cicada library
```

Comment:
Add the fact this can read multiple channels (compared with the IOCicadaProcessor) that reads only one.